### ISSUE TITLE
Stronger visual feedback for applicant when they are focused on a specific question

### ISSUE THREAD

**Issue by rockycodes** [civiform/civiform#5420]
**Is your feature request related to a problem? Please describe.**
Per feedback on [this pr](https://github.com/civiform/civiform/pull/5409), it is hard to tell which element is focused on. The current visual feedback is just a slight blue border.

**Describe the solution you'd like**
It should be visually clear to the user which element their cursor is focused on.

This issue is a follow-up to: https://github.com/civiform/civiform/issues/3571

**Comment by rockycodes** [comment]
This is covered by this issue which was created out of the accessibility audit: https://github.com/civiform/civiform/issues/5532
